### PR TITLE
Fix broken heading anchors pointing to admin login markdown page

### LINKED ISSUE #4837: Missing anchors when running mkdocs
**Describe the bug**
when running` mkdocs serve` locally I get these messages:

```
(base) chiararasi@n181-p119 scout % mkdocs servec
(base) chiararasi@n181-p119 scout % conda activate py39
(py39) chiararasi@n181-p119 scout % mkdocs serve
INFO    -  Building documentation...
INFO    -  Cleaning site directory
INFO    -  Doc file 'admin-guide/login-system.md' contains a link '#Google-OpenID-Connect-login-system', but there is no such anchor on this page.
INFO    -  Doc file 'admin-guide/login-system.md' contains a link '#Login-using-Lightweight-Directory-Access-Protocol', but there is no such anchor on this page.
INFO    -  Doc file 'admin-guide/setup-scout.md' contains a link './login-system.md/##Google-OpenID-Connect-login-system', but the doc 'admin-guide/login-system.md' does not contain an anchor
           '##Google-OpenID-Connect-login-system'.
INFO    -  Doc file 'admin-guide/setup-scout.md' contains a link './login-system.md/##Simple-login-with-userid-and-password', but the doc 'admin-guide/login-system.md' does not contain an anchor
           '##Simple-login-with-userid-and-password'.
INFO    -  Documentation built in 0.52 seconds
INFO    -  [16:48:43] Watching paths for changes: 'docs', 'mkdocs.yml'
INFO    -  [16:48:43] Serving on http://127.0.0.1:4000/scout/
```

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
 -Fix #4837 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
After the fix - no errors:

image

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,11 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ## [unreleased]
 ### Added
 - Button on SMN CN page to search variants within SMN1 and SMN2 genes
-- Options for selectively updating OMICS variants (fraser, outrider) on a case
 ### Changed
 - Documentation for OMICS variants and updating a case
+### Fixed
+- Options for selectively updating OMICS variants (fraser, outrider) on a case
```
Comment (dnil) [inline CHANGELOG.md:13]: Hm, this is still an added methinks: new options to the CLI.

File: docs/admin-guide/login-system.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Setting up a user login system
 
 Scout currently supports 3 types of login systems:
-  - [Google authentication via OpenID Connect](#Google-OpenID-Connect-login-system)
-  - [LDAP authentication](#Login-using-Lightweight-Directory-Access-Protocol)
+  - [Google authentication via OpenID Connect](#google-openid-connect-login-system)
```
Comment (dnil) [inline docs/admin-guide/login-system.md:4]: 💯

**Review by dnil — APPROVED** [review]
💯

File: CHANGELOG.md (hunk 1 above)
Comment (northwestwitch) [reply CHANGELOG.md]: moved to added!

**Pull request merged**